packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -1372,3 +1381,1205 @@ describe('when navbar menu items do not match given data fences', () => {
     });
   });
 });
+describe('With new navbar', () => {
```

Comment:
Is it on purpose to duplicate all the tests? 🤔

Reply:
Upon second thought, I removed the tests that are unrelated to the changes we're making.

Reply:
Thanks!